apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -708,7 +753,13 @@ func (m *ModelManagerAPI) fillInStatusBasedOnCloudCredentialValidity(ctx context
 		return errors.Trace(err)
```

Comment:
I think this entire function can get deleted? We did all of this hard work in the service https://github.com/ycliuhw/juju/blob/074b07bda4e163729d96a6015c9e57d9f92bd555/domain/model/service/modelservice.go#L247